pages/_en-US/ds-index/3ds-forwarders.md

```diff
@@ -80,17 +107,6 @@ PC:
 - **Linux users:** JavaFX. On Debian based systems run [this](https://gist.githubusercontent.com/puntillol59/7532b6583380baca236dcaf2d8f75b5c/raw/e8b9d193f8b24de941160c7292ec0bb3b997e98e/main.sh), or if you're on Arch run: `sudo pacman -S java8-openjfx && sudo archlinux-java set java-8-openjdk/jre`.
```

Comment:
```suggestion
- **Linux users:** JavaFX
   - Debian based: Run [this](https://gist.githubusercontent.com/puntillol59/7532b6583380baca236dcaf2d8f75b5c/raw/e8b9d193f8b24de941160c7292ec0bb3b997e98e/main.sh)
   - Arch: `sudo pacman -S java8-openjfx && sudo archlinux-java set java-8-openjdk/jre`
```
Oh, this wasn't changed here so might just commit this separately but I think this looks better while on the topic of changing this page.